Test Failure (20190416-0511): com.ibm.ws.microprofile.metrics.monitor_fat.TestEnableDisableFeaturesTest.testEDF10: java.lang.NoClassDefFoundError: javax.xml.ws.BindingProvider

RTC 262081
```
junit.framework.AssertionFailedError: 2019-04-16-15:16:13:408 [Unexpected FFDC reporting java.lang.NoClassDefFoundError was found (count = 1): C:\jazz-build-engines\jazz112.hursley.ibm.com\EBC.PROD.WASRTC\build\dev\image\output\wlp\usr\servers\TestEDF10\logs\ffdc\ffdc_19.04.16_15.16.09.0.log
>------Start of DE processing------ = [4/16/19, 15:16:09:173 BST]
>Exception = java.lang.NoClassDefFoundError
>Source = com.ibm.wsspi.injectionengine.MethodMap.getAllDeclaredMethods
>probeid = 106
>Stack Dump = java.lang.NoClassDefFoundError: javax.xml.ws.BindingProvider
> at java.base/java.lang.Class.getDeclaredMethodsImpl(Native Method)
> at java.base/java.lang.Class.getDeclaredMethods(Class.java:1126)
> at com.ibm.wsspi.injectionengine.MethodMap.getMethods(MethodMap.java:148)
> at com.ibm.wsspi.injectionengine.MethodMap.getAllDeclaredMethods(MethodMap.java:84)
> at com.ibm.ws.injectionengine.InjectionProcessorManager.getAllDeclaredMethods(InjectionProcessorManager.java:428)
> at com.ibm.ws.injectionengine.InjectionProcessorManager.processAnnotations(InjectionProcessorManager.java:230)
> at com.ibm.ws.injectionengine.AbstractInjectionEngine.processInjectionMetaData(AbstractInjectionEngine.java:513)
> at com.ibm.ws.injectionengine.osgi.internal.OSGiInjectionEngineImpl.processInjectionMetaData(OSGiInjectionEngineImpl.java:238)
> at com.ibm.ws.injectionengine.ReferenceContextImpl.processImpl(ReferenceContextImpl.java:588)
> at com.ibm.ws.injectionengine.ReferenceContextImpl.process(ReferenceContextImpl.java:287)
> at com.ibm.ws.injectionengine.osgi.internal.OSGiReferenceContextImpl.process(OSGiReferenceContextImpl.java:30)
> at com.ibm.ws.webcontainer.osgi.webapp.WebApp.commonInitializationStart(WebApp.java:253)
> at com.ibm.ws.webcontainer.webapp.WebApp.initialize(WebApp.java:979)
> at com.ibm.ws.webcontainer.webapp.WebApp.initialize(WebApp.java:6594)
> at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost.startWebApp(DynamicVirtualHost.java:467)
> at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost.startWebApplication(DynamicVirtualHost.java:462)
> at com.ibm.ws.webcontainer.osgi.WebContainer.startWebApplication(WebContainer.java:1140)
> at com.ibm.ws.webcontainer.osgi.WebContainer.access$000(WebContainer.java:110)
> at com.ibm.ws.webcontainer.osgi.WebContainer$3.run(WebContainer.java:952)
> at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:239)
> at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
> at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
> at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
> at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
> at java.base/java.lang.Thread.run(Thread.java:825)
>Caused by: java.lang.ClassNotFoundException: CWWKL0084W: The javax.xml.ws.BindingProvider class could not be loaded. Try enabling the jaxws-2.2 feature or a newer version of the feature in the server.xml file.
> at com.ibm.ws.classloading.internal.util.FeatureSuggestion.getExceptionWithSuggestion(FeatureSuggestion.java:103)
> at com.ibm.ws.classloading.internal.AppClassLoader.loadClass(AppClassLoader.java:491)
> at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:1032)
>Caused by: java.lang.ClassNotFoundException: javax.xml.ws.BindingProvider
> at com.ibm.ws.classloading.internal.AppClassLoader.findClassCommonLibraryClassLoaders(AppClassLoader.java:544)
> at com.ibm.ws.classloading.internal.AppClassLoader.findClass(AppClassLoader.java:286)
> at java.base/java.lang.ClassLoader.loadClassHelper(ClassLoader.java:1134)
> at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:1049)
> at com.ibm.ws.classloading.internal.AppClassLoader.findOrDelegateLoadClass(AppClassLoader.java:522)
> at com.ibm.ws.classloading.internal.AppClassLoader.loadClass(AppClassLoader.java:482)
]
at componenttest.custom.junit.runner.FATRunner.blowup(FATRunner.java:296)
at componenttest.custom.junit.runner.FATRunner.access$500(FATRunner.java:64)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:246)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:314)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:167)
```

--- a/dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/TestEnableDisableFeaturesTest.java
+++ b/dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/TestEnableDisableFeaturesTest.java
@@ -353,6 +353,7 @@ public class TestEnableDisableFeaturesTest {
     	Log.info(c, testName, "------- Remove JAX-WS application ------");
     	boolean rc1 = serverEDF10.removeAndStopDropinsApplications("testJaxWsApp.war");
     	Log.info(c, testName, "------- " + (rc1 ? "successfully removed" : "failed to remove") + " JAX-WS application ------");
+    	Assert.assertNotNull("CWWKT0017I NOT FOUND",serverEDF10.waitForStringInLogUsingMark(".*CWWKT0017I.*testJaxWsApp.*"));
     	serverEDF10.setMarkToEndOfLog();
     	serverEDF10.setServerConfigurationFile("server_noJaxWs.xml");
     	Assert.assertNotNull("CWWKF0008I NOT FOUND",serverEDF10.waitForStringInLogUsingMark("CWWKF0008I"));

